addons/resource.language.en_gb/resources/strings.po

```diff
@@ -7407,7 +7407,7 @@ msgstr ""
 
 #: xbmc/cores/AudioEngine/DSPAddons/ActiveAEDSP.cpp
 msgctxt "#15049"
-msgid "The audio DSP manager has been enabled without any enabled DSP add-on. Enable at least one add-on in order to use the audio DSP functionality."
+msgid "The audio DSP manager has been enabled without any DSP add-on. One enabled DSP add-on is required in order to use the audio DSP functionality."
```

Comment:
OK, now in line with PVR message exactly.

Its just that it sounds odd because of excessive use of enabled ```...has been enabled without any enabled... Enable at least...```